admin-server/src/main/resources/openapi/documentation.yaml

```diff
@@ -0,0 +1,20 @@
+openapi: "3.0.3"
```

Comment:
Why do we have openapi docs here?

Reply:
Won't we have it in stellar docs? I don't have a strong objection against it

Reply:
We should avoid serving the spec here. We don't do this for the platform server and it creates two sources of truth we need to maintain.

Reply:
I agree on that ^

Reply:
I will remove.